src/Components/Publishing/Sections/ArtworkCaption.tsx

```diff
@@ -196,11 +204,11 @@ export class ArtworkCaption extends React.Component<ArtworkCaptionProps, null> {
           sectionLayout={sectionLayout}
           className="display-artwork__caption"
         >
-          <ArtistName>{this.renderArtists()}</ArtistName>
-
+          <ArtistNames multipleNames={artists && artists.length > 1}>
+            {this.renderArtists()}
```

Comment:
Related to above, names is not super-specific, I think `hasMultipleArtists` is more explicit about the expectations